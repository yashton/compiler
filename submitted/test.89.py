s= "foo "OK" 'he is"