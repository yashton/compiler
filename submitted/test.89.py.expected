(ID "s")
(PUNCT "=")
(LIT "foo ")
(ID "OK")
(LIT " \'he is")
(ENDMARKER)
